docs/scripting-ref.rst

```diff
@@ -917,6 +917,84 @@ If your script returns the result of ``splash:png()`` in a top-level
 ``"png"`` key (as we've done in a previous example) then Splash UI
 will display it as an image.
 
+.. _splash-jpeg:
+
+splash:jpeg
+-----------
+
+Return a `width x height` screenshot of a current page in JPEG format.
+
+**Signature:** ``jpeg = splash:jpeg{width=nil, height=nil, render_all=false, scale_method='raster', quality=75}``
+
+**Parameters:**
+
+* width - optional, width of a screenshot in pixels;
+* height - optional, height of a screenshot in pixels;
+* render_all - optional, if ``true`` render the whole webpage;
+* scale_method - optional, method to use when resizing the image, ``'raster'``
+  or ``'vector'``
+* quality - optional, quality of JPEG image, integer in range from ``0`` to ``100``
+
+**Returns:** JPEG screenshot data.
+
+**Async:** no.
+
+Without arguments ``splash:jpeg()`` will take a snapshot of the current viewport.
```

Comment:
Yes, main reason why I just copy almost everything is that autocompletion shows all this documentation when you're using Splash with Jupyter.

I can probably remove two last blocks which start from ```If the result of ``splash:jpeg()`` ``` and just mention that it works the same as `splash:png()`. 

It's true that 4 parameters have almost the same documentation, but still there are some specifics for each format - like @kmike noticed extended height would be transparent for PNG and white for JPEG. I'm not sure we can completely omit this duplication problem, and while we have only 2 image formats - it should be fine to have some duplication in docs.

> One option is to provide something like splash:screenshot{format='jpeg'} function, and document splash:png and splash:jpeg only as shortcuts.

this method should support 2 different signatures - `quality` for jpeg only, also `transparency` for png only if we consider to add it (as discussed with @kmike) - wouldn't it be counterintuitive to have method with such dynamic signature? Documentation for this method will have to much ifs - like `use this parameter if format is 'jpeg'`. Also it would require much better input validation in lua than we have now.

`splash:screenshot` -> `splash:image{format='jpeg'}` as a shorter alternative